state_management/csharp/sdk/README.md

```diff
@@ -90,5 +89,7 @@ You're up and running! Both Dapr and your app logs will appear here.
 
 2. Stop and clean up application processes
 
+```bash
 dapr stop --app-id order-processor
-<!-- END_STEP -->
\ No newline at end of file
+```
+<!-- END_STEP -->
```

Comment:
Hi - I suggest we keep dapr stop out of the test step because it leads to timing issues.

Reply:
Hi @paulyuk 
How can I help for these small doc changes to be merged?

Reply:
Hi @farshaddavoudi - just need you to commit the two tweaks I asked for and I'll merge:
- please remove the END_STEP line on 95 so dapr stop is not included
- please edit the PR and retarget the base of release-1.13 branch

Then when you commit, tests should pass.  After that pls ping and I'll merge!

Reply:
@paulyuk I've done the tweaks you requested. Anything else to do?